src/System.IO.Pipelines/ref/Configurations.props

```diff
@@ -1,11 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard1.3;
-    </PackageConfigurations>
     <BuildConfigurations>
-      $(PackageConfigurations);
       netstandard;
```

Comment:
This now adds the netstandard2.0 configuration to the package which didn't exist before we should decide if we want to do that or not.

Reply:
The src already has a netstandard PackageConfiguration and since the ref was being built for ns1.3 it was alread ns2.0 supported. What configuration should we build it for instead?

Reply:
I think I'm OK with adding the netstandard2.0 configuration to the package. It is probably better that way anyhow because it helps eliminate some package dependencies.